cli.js

```diff
@@ -15,11 +15,15 @@ const cli = meow(`
 	Options
 	  --upload, -u   Measure upload speed in addition to download speed
 	  --single-line  Reduce spacing and output to a single line
+	  --json         Process output in json format also forces single line 
```

Comment:
```suggestion
	  --json         JSON output
```

You also need to update the readme.